### ISSUE TITLE
Please add this screenshot amd instructions to documenation (running scripts in Termux)

### ISSUE THREAD

**Issue by sudomain** [keymapperorg/KeyMapper#1189]
Hi,

I'm not sure where this would go in the docs, but scripts in Termux can be executed by following the permission granting instructions [here](https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent) and using a Key Mapper `send intent` action with the following settings:
![Screenshot_20230727-161027](https://github.com/keymapperorg/KeyMapper/assets/30786326/c40915fc-bb8b-4d2b-8c79-c398ca82dbc3)

I only mention it here because the service, package, and class names are not obvious based on the current docs.

**Comment by Codereamp** [comment]
I agree that either screenshots or a text description would be nice. In my case I also needed running a shell script silent and background so additionally to the extra field visible here I had a boolean field com.termux.RUN_COMMAND_BACKGROUND

**Comment by Codereamp** [comment]
Also I little praise torwards the app. The developer might find it more appropriate in other places if he wishes, sorry if this is the case :) On Nook devices there are four hardware buttons. Given that KM allows three (press, long, double) actions per key and even sequence, we would possible have at least 12 actions available system wise. Yes, mostly they already used for paging in readers, but nevertheless having such a wide range of choices are just great.  Thanks

**Comment by jambl3r** [comment]
replaced by shell action
